backend/src/models/classroom.ts

```diff
@@ -21,16 +21,19 @@ Classroom.init(
     name: {
       type: DataTypes.CITEXT,
       allowNull: false,
-      unique: true,
-      validate: { notEmpty: { msg: 'Classroom name can not be empty' } },
+      unique: {
+        name: 'ClassroomNameUniqueValidation',
```

Comment:
Do these new unique constrains need to be added through migrations?

Reply:
As far as I know, no. I know that constraints are set in migrations because they are defined at SQL level and migrations determine the DB schema but unique was already added as a constraint through migration for this model. That didn't change, only thing that is changed is unique constraint value, in this case name and error message are added and I think that is important for Sequelize only. Database is still going to check if unique constraint is broken, if it is database will throw an error and Sequelize will forward this error with this message.